### ISSUE TITLE
Onsite SEO - Lesson Plan

### ISSUE THREAD

**Issue by courtneyr-dev** [WordPress/Learn#599]
# Details

- Topic description: What can users do to improve their SEO? (Keywords in headings, titled images with alt text, excerpt description, use of schema. You can suggest plugins that help check the key areas.  What beyond a post/page should be considered? Categories, tags, page load time, etc.
- Audience (User, Developer, Contributor, etc.): Users
- Learning objectives: 
- Content type (Course, Tutorial, Online Workshop, or Lesson Plan): Lesson Plan
- WordPress version (optional): 
- Will you be creating this content? (Yes or No): 

# Related Resources

# Next steps for SMEs
Please follow the team handbook "[Vetting topic ideas](https://make.wordpress.org/training/handbook/training-team-how-to-guides/vetting-topic-ideas/)" to vet this topic.

_Issue update by @bsanevans to match current topic vetting template._
